modules/infra/cli/port.c

```diff
@@ -24,12 +25,28 @@ static void port_show(const struct gr_api_client *, const struct gr_iface *iface
 	printf("n_txq: %u\n", port->n_txq);
 	printf("rxq_size: %u\n", port->rxq_size);
 	printf("txq_size: %u\n", port->txq_size);
+
+	if (iface->mode == GR_IFACE_MODE_L1_XC) {
+		if (iface_from_id(c, iface->domain_id, &peer) == 0)
+			printf("xc peer: %s\n", peer.name);
+		else
+			printf("xc peer: unknown\n");
```

Comment:
cosmetics:

s/xc peer/xc_peer/

And if iface_from_id returns an error, just print the domain_id as is.